docs/topics/eclipse-installing-plugin.adoc

```diff
@@ -15,11 +15,11 @@ ifdef::disconnected[]
 You can install the {PluginName} in a disconnected network environment.
 endif::[]
 
-The {PluginName} has been tested with the Eclipse IDE for Java Enterprise Developers 2020-09 R and Red Hat CodeReady Studio 12.15.
+The {PluginName} has been tested with the Eclipse IDE for Java Enterprise Developers 2021-03 and Red Hat CodeReady Studio 12.19.1.
 
 .Prerequisites
 
-* link:{CodeReadyStudioDownloadPageURL}[Red Hat CodeReady Studio] _or_ link:https://www.eclipse.org/downloads/packages/release/2020-09/r/eclipse-ide-enterprise-java-developers[Eclipse IDE for Java Enterprise Developers 2020-09 R]
+* link:{CodeReadyStudioDownloadPageURL}[Red Hat CodeReady Studio] _or_ link:https://www.eclipse.org/downloads/packages/release/2021-03/r/eclipse-ide-java-developers[Eclipse IDE for Java Enterprise Developers 2021-03].
```

Comment:
The Eclipse IDE link does not actually point to 2021-03. It resolves to the latest version (2021-06). That means you should probably change the link to the general Eclipse IDE for JED URL and add a short procedure (Click the 2021-03 link...).